### PR TITLE
issue #237 spin fix.

### PR DESCRIPTION
**Pull request by pulquero**
This PR addresses GitHub issue: #237 

Briefly describe the changes proposed in this PR:

- Fixed spin bug

Make sure you've followed the [Contributor Guidelines](https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md). In particular (please tick to indicate you've taken care of it):

- [x] RDF4J code formatting has been applied
- [x] tests are included
- [x] all tests succeed

Signed-off-by: Mark Hale <mark.hale@physics.org>

### CODE REVIEW

File: core/queryalgebra/model/src/main/java/org/eclipse/rdf4j/query/algebra/Slice.java [hunk 1]
```diff
@@ -29,10 +29,11 @@
 	 *--------------*/
 
 	public Slice() {
+		this(null);
 	}
 
 	public Slice(TupleExpr arg) {
-		this(arg, 0, -1);
+		this(arg, -1, -1);
```
Comment (pulquero) [inline core/queryalgebra/model/src/main/java/org/eclipse/rdf4j/query/algebra/Slice.java]: Changes to fix default (unset) value to -1. This is the same as that used by TupleExprBuilder and ensures equals() works properly no matter which constructor is used.

**Comment by pulquero** [comment]
Can anyone help with the conflicts, I don't understand what the problem is.

**Comment by pulquero** [comment]
BTW, is there any reason why travis is not being used?

**Comment by jeenbroekstra** [comment]
@pulquero I'm not sure why you ask about Travis. We have a Hudson CI in place at https://hudson.eclipse.org/rdf4j/ - I thought you knew?

**Comment by pulquero** [comment]
All fixed, my remotes werent setup properly.
Travis can build pull requests which isnt the same thing.

**Comment by jeenbroekstra** [comment]
I'm not really familiar with Travis but if you feel it would be useful, flick an e-mail to rdf4j-dev so we can discuss it.

**Comment by jeenbroekstra** [comment]
Anyway, approved, merging.

**Pull request merged**